Complete overhaul to the tracker

TheLX5 opened the pull request:
This PR has the purpose to redesign and adapt the tracker for easier usage and include compatibility with the upcoming SMW v2 AP implementation.

Changes:
* Maps are now in a single image under a single tab
* There's a level image for every single level in the game
  * The levels have exact coordinates for every kind of check supported in them
* Added support for several SMW v2 features
  * Blocksanity
  * 3-Up Moons
  * Hidden 1-Ups
  * Bonus Blocks prizes
  * Special Zone Clear

TheLX5 committed:
Initial v2.0.0 implementation

PoryGone commented on super_mario_world_randomizer_porygone/manifest.json:
```diff
@@ -2,9 +2,9 @@
     "game_name": "Super Mario World",
     "platform": "Super Nintendo Entertainment System",
-    "package_version": "1.0.1",
+    "package_version": "2.0.0",
```
```suggestion
    "package_version": "2.0.0b1",
```
I prefer to mark pre-release versions as incremental betas until the AP side gets merged and things stabilize.

TheLX5 replied:
Ah, alright.

PoryGone commented:
Not a huge deal, but is there a reason that the subfolders of the `locations` folder were removed?

TheLX5 committed:
Update super_mario_world_randomizer_porygone/manifest.json

Co-authored-by: PoryGone <98504756+PoryGone@users.noreply.github.com>

TheLX5 commented:
> Not a huge deal, but is there a reason that the subfolders of the `locations` folder were removed?

My script just placed everything into the root folder, didn't bother with organization too much.

TheLX5 committed:
Swapped exits in Forest Ghost House

TheLX5 committed:
Added a missing Hidden 1-Up location in Forest of Illusion 3

TheLX5 committed:
Updated Dragon coins access rules in Vanilla Dome 1

TheLX5 committed:
Removed unused files

TheLX5 committed:
Update Donut Plains 2's secret exit access rules

TheLX5 committed:
Update Donut Plains 3's Dragon coins access rules

TheLX5 committed:
Changed Star World instances to Star Road to match the apworld's names

TheLX5 committed:
Update map.json

TheLX5 committed:
Updated blocksanity's AP IDs & reflect previous commits name changes

TheLX5 commented:
Added fixes for the things you reported to me via DMs. Also changed "Star World" instances to "Star Road" to match the apworld names.

I haven't gotten around updating the logic rules for the GSP wall in DP1, so I'm not gonna add those to the tracker yet.

TheLX5 committed:
Mix CI2 sublevels in single images

TheLX5 committed:
Made DP1 GSP wall only require GSP and Cape

TheLX5 committed:
Batch of changes

TheLX5 committed:
Fix mismatch of Room-Goal names

TheLX5 committed:
Add "out-of-logic" logic for every possible check

TheLX5 committed:
Fix YI4's last Dragon Coin position
